app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -259,9 +266,27 @@ private fun onBindCustomHomeTile(uiLifecycleCancelJob: Job, holder: TileViewHold
         }
 
         // Wait for both to complete so we can animate them together.
-        iconView.setImageBitmap(screenshot.await()) // TODO: if null, provide placeholder.
+        val screenshotToRound = screenshot.await() // TODO: if null, provide placeholder.
+        val cornerRadius = 8.toFloat()
+        // Need copy of screenshot bitmap because it is immutable
+        val copyOfScreenshot = Bitmap.createBitmap(screenshotToRound.width, screenshotToRound.height, screenshotToRound.config)
```

Comment:
Please make this a helper function – the code gets really hard to read when there are many low-level methods.

I think this should be an extension function: `Bitmap.withRoundedCorners(cornerRadius: Float)`